x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicators_table/cell_renderer.tsx

```diff
@@ -12,13 +12,13 @@ import React from 'react';
 import { useKibana } from '../../../../hooks/use_kibana';
 import { EMPTY_VALUE } from '../../../../../common/constants';
 import { Indicator } from '../../../../../common/types/indicator';
-import { displayValue } from '../../lib/display_value';
+import { getDisplayName } from '../../lib/display_value';
 import { IndicatorField } from '../indicator_field/indicator_field';
 import { IndicatorsTableContext } from './context';
 import { ActionsRowCell } from './actions_row_cell';
 
 export enum ComputedIndicatorFieldId {
-  DisplayValue = 'display_value',
+  DisplayName = 'display_name',
```

Comment:
updating `DisplayValue` to the `DisplayName`, I think this is the term we are using now. DisplayName has a corresponding field in document and a value until we don't have it incorporated in the ECS schema